### PR TITLE
Make travis work

### PR DESCRIPTION
**Pull request by kotoko**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@ language: java
 jdk:
   - oraclejdk8
 script:
+  - cd api/
   - ./gradlew check
-  - ./gradlew integrationTest
   - ./gradlew jacocoTestReport
```
Comment (michalkowol) [inline .travis.yml]: Could you please add building frontend as well?

- `yarn`

Comment (kotoko) [reply .travis.yml]: Sure.

**Comment by kotoko** [comment]
I added testing building ui (works) and testing api (partially works). I managed to setup postgres with travis. I had to change build.gradle to read variables from environment. Now there is ugly if/else block, but works.

There are errors in build log related to kotlin(?):
```
(more errors)
e: /home/travis/build/codeloopeu/thehub/api/src/it/kotlin/com/softwareberg/thehub/jobs/DatabaseOperationsTest.kt: (69, 46): Unresolved reference: COMPANIES
e: /home/travis/build/codeloopeu/thehub/api/src/it/kotlin/com/softwareberg/thehub/jobs/DatabaseOperationsTest.kt: (91, 35): Unresolved reference: COMPANIES
e: /home/travis/build/codeloopeu/thehub/api/src/it/kotlin/com/softwareberg/thehub/jobs/DatabaseOperationsTest.kt: (91, 46): Unresolved reference: COMPANIES
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':compileIntegrationTestKotlin'.
> Compilation error. See log for more details
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
BUILD FAILED in 1m 51s
14 actionable tasks: 14 executed
```


We also have nice 2 tasks when you click on travis details (inspired: https://stackoverflow.com/a/44054333).

**Comment by kotoko** [comment]
Travis seems to be working now.

**Comment by kotoko** [comment]
Do we want to run `./gradlew integrationTest`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      before_install:
+        - sudo /etc/init.d/postgresql stop
+        - cd api/docker/
+        - docker-compose up -d
+        - cd ../../
+      script:
+        - cd api/
+        - ./gradlew check
+        - ./gradlew jacocoTestReport
+
```
Comment (michalkowol) [inline .travis.yml]: - [ ] remove empty line

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    - language: node_js
+      node_js:
+        - "10.8.0"
+      script:
+        - cd ui/
+        - yarn install
+
```
Comment (michalkowol) [inline .travis.yml]: - [ ] remove empty line

File: README.md [hunk 4]
```diff
@@ -15,10 +15,21 @@ http POST http://localhost:8080/api/jobs/sync
 open http://localhost:8080/api/jobs
 ```
 
+We are using [jOOQ](https://www.jooq.org) for generating code that deals with SQL. If you change something related to databse you will need to regenerate this code. You can generate it explicitly:
```
Comment (michalkowol) [inline README.md]: - [ ] `We are using` -> `We use`

**Pull request merged**